2021/d01/p1.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+from typing import Iterator
+
+INPUT_FILE_PATH = Path('..', 'inputs', '1.txt')
+
+
+def get_measurements_from_input(input_text: str) -> list:
+    """
+    :param input_text: input test to process
+    :return: split measurements from the input
+    """
+    return list(map(int, input_text.splitlines()))
+
+
+def get_depth_measurements_increases(measurements: Iterator[int]) -> int:
```

Comment:
You annotate the argument as an iterator but use indexing. Should be list or sequence.